baselines/dom.generated.d.ts

```diff
@@ -8445,9 +8456,7 @@ declare var IDBCursorWithValue: {
 };
 
 interface IDBDatabaseEventMap {
-    "abort": Event;
     "close": Event;
-    "error": Event;
```

Comment:
This cannot be inferred without manually linking the affected ones to the bubbling events. I'm not going to do that, at least for now.